assets/scss/styles.scss

```diff
@@ -21,3 +20,7 @@
 @import './components/_phase-banner';
 @import './components/_input-file';
 @import './components/skip-link';
+
+@tailwind components;
```

Comment:
Maybe these should get moved to an app.css which would also be the custom styles for the app.

This would make a standard place to add your styles + turn off components as per the user needs.

Reply:
Let me know what you think of the changes I just pushed related to this